test/Engine/ProtoTest/Associative/BuiltinMethodsTest.cs

```diff
@@ -1069,6 +1069,23 @@ public void TestTryGetValuesFromDictionary08()
            thisTest.Verify("r", null);
        }
 
+        [Test]
+        [Category("Failure")]
+        // To-Do as this use case is failing, re-visit and make sure this test case passes.  
+        public void TestTryGetValuesFromDictionary09()
+        {
+            string code = @"
+a = { ""in"" : 42, ""out"" : 24 };
+b = { ""in"" : 24, ""out"" : 42 };
+c = [[a],[[b]]];
```

Comment:
Nested Dictionaries work fine but fails when the level of nesting for list "a" i.e 1, is less than the nesting level of list "b" i.e 2. However if C is defined as c = [a,[[b]]], the output is correct.